npm/parseSelector.go

```diff
@@ -121,73 +121,170 @@ func sortSelector(selector *metav1.LabelSelector) {
 	selector.MatchExpressions = sortedReqs
 }
 
+// getSetNameForMultiValueSelector takes in label with multiple values without operator
+// and returns a new 2nd level ipset name
+func getSetNameForMultiValueSelector(key string, vals []string) string {
+	rtStr := key
```

Comment:
What is "rtStr"?

Reply:
rtrStr = returnString. Corrected the variableName

Reply:
@vakalapa `returnString` is not clearer; in Go it is better to name the variable by what it holds, not by its type or use - "~returnString~ is the ... (`IPSetName` maybe?)"
maybe that is what @JungukCho was getting at